### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by johnprobable**
Adding reminder for the --main_config flag to the cloud launch section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ To start a deployment in the cloud:
 $ spatial alpha cloud launch -d <your-deployment-name> -a <your-assembly-name>
 ```
 
+As above, you can run these commands from the /SpatialOS directory or use the `--main_config=\<path to spatialos.json\>` flag to point to your project config when running the commands from another directory.
```
Comment (JonasImprobable) [inline README.md]: Nit: formatting:
```suggestion
As above, you can run these commands from the `SpatialOS` directory or use the `--main_config=<path to your spatialos.json>` flag to point to your project configuration file when running the commands from another directory.
```

Comment (johnprobable) [reply README.md]: Fixed (and applied the nit to the line I copied from the local launch section.)

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ To launch a local instance of SpatialOS running the project, run
 ```bash
 $ spatial alpha local launch
 ```
-from the SpatialOS directory (or from any location by adding the `--main_config=\<path to spatialos.json\>` flag). This starts SpatialOS locally and runs the server workers `HelloWorker` and `DiceWorker`.
+from the `SpatialOS` directory (or from any location by adding the `--main_config=\<path to your spatialos.json\>` flag). This starts SpatialOS locally and runs the server workers `HelloWorker` and `DiceWorker`.
```
Comment (JonasImprobable) [inline README.md]:
```suggestion
from the `SpatialOS` directory (or from any location by adding the `--main_config=<path to your spatialos.json>` flag). This starts SpatialOS locally and runs the server workers `HelloWorker` and `DiceWorker`.
```

File: README.md [hunk 3]
```diff
@@ -77,6 +77,8 @@ To start a deployment in the cloud:
 $ spatial alpha cloud launch -d <your-deployment-name> -a <your-assembly-name>
 ```
 
+As above, you can run these commands from the `SpatialOS` directory or use the `--main_config=\<path to your spatialos.json\>` flag to point to your project config when running the commands from another directory.
```
Comment (JonasImprobable) [inline README.md]:
```suggestion
As above, you can run these commands from the `SpatialOS` directory or use the `--main_config=/<path to your spatialos.json>` flag to point to your project config when running the commands from another directory.
```

**Pull request merged**